.github/workflows/check-code-quality.yml

```
@@ -75,3 +75,17 @@ jobs:
             git status --porcelain --untracked-files=no
             exit 1
           fi
+      
+      - name: Check generated code
+        if: success()
+        run: |
+          pip install -r code_generation/requirements.txt
+          python code_generation/code_gen.py
+          if [ -n "$(git status --porcelain)" ]; then
+            echo "The following files are outdated or were manually updated:"
```

maybe nitpick suggestion (please verify that it works)
```suggestion
            echo "\nError: The following files are outdated or were manually updated:"
```

---

That does not work, but simply adding and empty `echo` before hand does the trick.